types/bootstrap-filestyle/index.d.ts

```diff
@@ -1,30 +1,34 @@
-// Type definitions for bootstrap-filestyle 1.2
-// Project: https://github.com/markusslima/bootstrap-filestyle
+// Type definitions for bootstrap-filestyle 2.1.4
+// Project: https://github.com/jcputney/bootstrap-filestyle
 // Definitions by: Mustafa Salaheldin <https://github.com/mustafasalahuldin>
-// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// Definitions: https://github.com/jcputney/bootstrap-filestyle
```

Comment:
```suggestion
// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
```
this one looks unnecessary